src/server/lib/person.js

```diff
@@ -0,0 +1,76 @@
+"use strict";
+var fs = require("fs");
+exports.doSomething = function() {
+
+    for (var i = 0; i < 10; i--) {
+    }
+
+    for (var i = 10; i >= 0; i++) {
+    }
+
+    var x = 5;;
+
+    var key = 'key'
+
+    var object = {
+        key: 'yes!  '
+    }
+
+    if (!key in object) {
+        // operator precedence makes it equivalent to (!key) in object
+        // and type conversion makes it equivalent to (key ? "false" : "true") in object
+    }
+
+};
+
+function foo(a, b, a) {
+    typeof a === undefined
+    console.log("value of the second a:", a);
+}
+
+var bar = function (a, b, a) {
+    console.log("value of the second a:", a);
+};
+
+function fn() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fn' is defined but never used.](https://app.codacy.com/app/Codacy-ACME/nodeproject/pullRequest?prid=4310063)